src/main/java/io/confluent/connect/hdfs/partitioner/TimeBasedPartitioner.java

```diff
@@ -86,16 +89,53 @@ public void configure(Map<String, Object> config) {
     String hiveIntString = (String) config.get(HdfsSinkConnectorConfig.HIVE_INTEGRATION_CONFIG);
     boolean hiveIntegration = hiveIntString != null && hiveIntString.toLowerCase().equals("true");
 
+    timeFieldName = (String) config.get(HdfsSinkConnectorConfig.PARTITION_TIME_FIELD_NAME_CONFIG);
+
```

Comment:
How about checking this field name and setting it to null if the field name is empty or is blank when trimmed? That way the field name is either null or contains a name.